src/modules/frost/bench_impl.h

```diff
@@ -0,0 +1,221 @@
+/***********************************************************************
+ * Copyright (c) 2025 Bank of Italy                                    *
+ * Distributed under the MIT software license, see the accompanying    *
+ * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
+ ***********************************************************************/
+
+#ifndef SECP256K1_MODULE_FROST_BENCH_H
+#define SECP256K1_MODULE_FROST_BENCH_H
+
+#include "../../../include/secp256k1_frost.h"
+#include "fill_random.h"
+
+#define MSGLEN_FROST_BENCH (32)
+#define BENCH_FROST_MAX_PARTICIPANTS 3
+#define BENCH_FROST_THR_PARTICIPANTS 2
+
+typedef struct {
+  secp256k1_frost_keypair *keypairs;
+  secp256k1_frost_nonce **nonces;
+  secp256k1_frost_nonce_commitment *signing_commitments;
+  secp256k1_frost_pubkey *public_keys;
+  secp256k1_frost_signature_share *signature_shares;
+  unsigned char signature[64];
+} secp256k1_frost_data_per_iter;
```

Comment:
`signature` was 65 bytes on the standard compliant version. This PR applies the benchmark on a varsion that is only BIP340-compliant, so signature has been resized to 64.

**TODO**: when `ENABLE_MODULE_FROST_BIP340_MODE`  will be introduced in a later PR, conditionalize this statement to it.